ui/src/styles.scss

```diff
@@ -280,23 +284,27 @@ body {
   }
 }
 
-.sidenav-conversionByTable{
-  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%);
+.sidenav-conversionByTable {
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%),
+    0px 1px 3px 0px rgb(0 0 0 / 12%);
 }
 
-.sidenav-conversionByPercentage{
-  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%);
+.sidenav-conversionByPercentage {
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%),
+    0px 1px 3px 0px rgb(0 0 0 / 12%);
 }
 
-.sidenav-databaseDefinitions{
-  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%);
+.sidenav-databaseDefinitions {
+  box-shadow: 0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%),
+    0px 1px 3px 0px rgb(0 0 0 / 12%);
 }
 
-
-.sidenav-title{
-  margin-top :10px;
+.sidenav-title {
+  margin-top: 10px;
   margin-bottom: 0px;
 }
+<<<<<<< HEAD
+=======
```

Comment:
Please resolve the merge conflicts before checking the code in or marking the PR ready for review.

Reply:
Will take care of this.